src/main/archetype/all/pom.xml

```diff
@@ -0,0 +1,196 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- ====================================================================== -->
+    <!-- P A R E N T  P R O J E C T  D E S C R I P T I O N                      -->
+    <!-- ====================================================================== -->
+    <parent>
+        <groupId>${groupId}</groupId>
+        <artifactId>${rootArtifactId}</artifactId>
+        <version>${version}</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <!-- ====================================================================== -->
+    <!-- P R O J E C T  D E S C R I P T I O N                                   -->
+    <!-- ====================================================================== -->
+    <artifactId>${artifactId}</artifactId>
+    <packaging>content-package</packaging>
+    <name>${artifactName} - All</name>
+    <description>All content package for ${artifactName}</description>
+
+    <!-- ====================================================================== -->
+    <!-- B U I L D   D E F I N I T I O N                                        -->
+    <!-- ====================================================================== -->
+    <build>
+        <plugins>
+            <!-- ====================================================================== -->
+            <!-- V A U L T   P A C K A G E   P L U G I N S                              -->
+            <!-- ====================================================================== -->
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <group>${packageGroup}</group>
+                    <embeddeds>
+                        <embedded>
+                            <groupId>${groupId}</groupId>
+                            <artifactId>${rootArtifactId}.ui.apps</artifactId>
+                            <type>zip</type>
+                            <target>/apps/${appsFolderName}-packages/application/install</target>
+                        </embedded>
+                        <embedded>
+                            <groupId>${groupId}</groupId>
+                            <artifactId>${rootArtifactId}.ui.content</artifactId>
+                            <type>zip</type>
+                            <target>/apps/${appsFolderName}-packages/content/install</target>
+                        </embedded>
+                        <embedded>
+                            <groupId>com.adobe.cq</groupId>
+                            <artifactId>core.wcm.components.all</artifactId>
+                            <type>zip</type>
+                            <target>/apps/vendor-packages/application/install</target>
+                        </embedded>
+#if ( $optionIncludeExamples == "y" )
+                        <embedded>
```

Comment:
again, this appears to be mixed package - i believe you would need to again be split into embeddeds for the Core and Content packages for examples

Reply:
This should not be used in production, I believe it's ok to leave it as is for local development.